services/ocl-loader/pom.xml

```diff
@@ -40,6 +40,17 @@
             <version>2.14.0-rc1</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.3</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
Slf4j impl should be only in test scope. The runtime will décides of the impl/binding to use.

Reply:
Not necessary. It depends of the use they will do: some plugins might want, some others don't. Having a convenient method in the ocl loader service is ok.

Reply:
Ok, it makes sense.